### PR TITLE
Fix error when copying sheets with certain image anchors

### LINKED ISSUE #574: Error when copying sheet's with images with MoveAndSize placement.
## Read and complete the full issue template

**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [ ] Feature

**Version of ClosedXML**

e.g. 0.90.0

**What is the current behavior?**

When copying sheets with images with certain anchors, the following exception is thrown:
"To set the width, the placement should be FreeFloating or Move"

**What is the expected behavior or new feature?**

Copy sheets as expected.

**Did this work in previous versions of our tool?  Which versions?**

Probably not.

## Reproducibility
**This is an important section. Read it carefully. Failure to do so will cause a 'RTFM' comment.**

Without a code sample, it is unlikely that your issue will get attention. Don't be lazy. Do the effort and assist the developers to reproduce your problem. Code samples should be [minimal complete and verifiable](https://stackoverflow.com/help/mcve). Sample spreadsheets should be attached whenever applicable. Remove sensitive information.

**Code to reproduce problem:**
```c#
        [Test]
        public void CanCopySheetsWithAllAnchorTypes()
        {
            using (var stream = TestHelper.GetStreamFromResource(TestHelper.GetResourcePath(@"Examples\ImageHandling\ImageAnchors.xlsx")))
            using (var wb = new XLWorkbook(stream))
            {
                var ws = wb.Worksheets.First();
                ws.CopyTo("Copy1");

                var ws2 = wb.Worksheets.Skip(1).First();
                ws2.CopyTo("Copy2");

                var ws3 = wb.Worksheets.Skip(2).First();
                ws3.CopyTo("Copy3");

                var ws4 = wb.Worksheets.Skip(3).First();
                ws3.CopyTo("Copy4");
            }
        }

```
- [ ] I attached a sample spreadsheet.  (You can drag files on to this issue)

### PR DESCRIPTION
**Pull request by bsermons**
#### What's this PR do?
Fixes #574 
Allows a sheet to be copied that contains images with MoveAndSize placement.  It just temporarily sets the image placement to something that allows width/height to be set while copying excel sheets.

#### Any background context you want to provide?
This was the easiest thing to do to fix the issue.  If you decide the width/height setters should not have that placement constraint then I'll be glad to change the PR to do that instead.

#### Questions:
- Is there a blog post? no
- Does the knowledge base need an update? no
- Does this add new (C#) dependencies? no

- [ ] C# Code Review: @csreviewer
- [ ] Test Automation Review: @csreviewer

### CODE REVIEW

File: ClosedXML/Excel/XLWorksheet.cs [hunk 1]
```diff
@@ -584,8 +584,9 @@ public IXLWorksheet CopyTo(XLWorkbook workbook, String newSheetName, Int32 posit
             foreach (var picture in Pictures)
             {
                 var newPic = targetSheet.AddPicture(picture.ImageStream, picture.Format, picture.Name)
-                    .WithPlacement(picture.Placement)
-                    .WithSize(picture.Width, picture.Height);
+                    .WithPlacement(XLPicturePlacement.FreeFloating)
```
Comment (igitur) [inline ClosedXML/Excel/XLWorksheet.cs]: Isn't it sufficient to have only 2 lines? With Size first, and placement second?

File: ClosedXML_Tests/Excel/Loading/LoadingTests.cs [hunk 2]
```diff
@@ -148,6 +148,26 @@ public void CanLoadFileWithImagesWithCorrectAnchorTypes()
         }
 
         [Test]
+        public void CanCopySheetsWithAllAnchorTypes()
```
Comment (igitur) [inline ClosedXML_Tests/Excel/Loading/LoadingTests.cs]: This isn't really a loading test. Please move this test to `XLWorksheetTests`.

File: ClosedXML/Excel/XLWorksheet.cs (hunk 1 above)
Comment (bsermons) [reply ClosedXML/Excel/XLWorksheet.cs]: Sorry, I got busy and missed your comments.  The problem is that there is an image with the `MoveAndSize` placement type saved in that excel file.  Trying to set the size in `WithSize` throws exceptions for images with this anchor type. See here: https://github.com/ClosedXML/ClosedXML/blob/230236e6ac43ff27c9947c966dd9cd1bc23ee06c/ClosedXML/Excel/Drawings/XLPicture.cs#L223

This PR just sets the placement first to something less restrictive on sizing and then resets it afterwards to the original anchor type.  The other alternative I see would be to remove the lines that throw exceptions in XLPicture.  You might have a better idea of how to fix it that I'm not aware of though.

Comment (igitur) [reply ClosedXML/Excel/XLWorksheet.cs]: Ok, I'm happy with this.

File: ClosedXML_Tests/Excel/Loading/LoadingTests.cs (hunk 2 above)
Comment (igitur) [reply ClosedXML_Tests/Excel/Loading/LoadingTests.cs]: Please do this, then I'll merge.

**Comment by bsermons** [comment]
Made the changes you requested.  Let me know if you need anything else.

**Pull request merged**